packages/common/src/eips/4200.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "EIP-4200",
+  "number": 4200,
+  "comment": "Static relative jumps",
+  "url": "https://eips.ethereum.org/EIPS/eip-4200",
+  "status": "Review",
+  "minimumHardfork": "berlin",
+  "requiredEIPs": [3540, 3670],
```

Comment:
Short nit: since we have a transitive relation here I would (generally) have a tendency here to keep this to the minimal set and just add 3670. So 3540 will be automatically checked along 3670 activation, and if we add both here we unnecessarily double the 3540 check.

Reply:
I understand your point, but I wanted to be consistent with the EIP specification. At the time that I wrote this code, it still requires both of those (yes, this is a transitive / circular relation). I would say, in the first iteration of implementing Big EOF, lets try to be as close to the EIP spec, and we can optimize/update this later (for now, we should ensure that it works, it looks clean-ish, and it passes the tests (either generated by us, or what would be more lovely - ethereum/tests unit tests))